lib/dotcom_web/templates/layout/root.html.heex

```diff
@@ -76,6 +76,21 @@
           </div>
         </div>
       <% end %>
+      <!-- START BANNERS -->
+      <div
+        id="mobile-app-banner"
+        style="background: #DCD3E8; display: none; margin: 0; padding: 10px; text-align: center"
+      >
+        <a
+          href="https://mbta.com/app-store?pt=117998862&ct=Dotcom%20Banners&mt=8&referrer=utm_source%3Ddotcom%26utm_campaign%3Dbanners"
+          rel="noreferrer"
+          style="color: black; display: block"
+          target="_blank"
+        >
+          Download <strong>MBTA Go</strong> to <span></span> &#x2192;
```

Comment:
```suggestion
          Download <strong>MBTA Go</strong> to <span>track your trip</span> &#x2192;
```
Let's offer a fallback that'll be adjusted by the JS. Just in case the script doesn't connect.